flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionManagerTest.java

```diff
@@ -55,6 +68,29 @@ public void testCreateViewForRegisteredPartition() throws Exception {
                 partition.getPartitionId(), 0, new NoOpBufferAvailablityListener());
     }
 
+    @Test
+    public void testCreateViewNotifierAfterRegisteredPartition() throws Exception {
+        final ResultPartitionManager partitionManager = new ResultPartitionManager();
+        final ResultPartition partition = createPartition();
+
+        partitionManager.registerResultPartition(partition);
+        PartitionRequestNotifier partitionRequestNotifier = new TestingPartitionRequestNotifier();
+        assertNotNull(partitionManager.createSubpartitionViewOrNotify(
+                partition.getPartitionId(), 0, new NoOpBufferAvailablityListener(), partitionRequestNotifier));
```

Comment:
instead of `assertNotNull`, you should:
1. check that list of registered listeners/notifiers in the `ResultPartitionManager` is empty
2. assert that `partitionRequestNotifier` has been notified with correct partition and subpartition id.

Reply:
> assert that partitionRequestNotifier has been notified with correct partition and subpartition id. 

has this been added? I don't see it in this test 🤔